### PR TITLE
BLD,RLS: jinja_tornado v0.1.0 packaging

### PR DESCRIPTION
**Pull request by westurner**
- See also #1

### CODE REVIEW

**Comment by thkang2** [comment]
Thank you! I'm not really into python development for some time, maybe you
can fork this and do anything as you wish!

2015-12-03 17:59 GMT+09:00 Wes Turner notifications@github.com:

**Comment by westurner** [comment]
Thanks again!

Should I namespace it differently (e.g. mv the directory (losing the git
history)) or would it be alright if I register this (w/ some CLNup and DOC
changes) as 'jinja_tornado'?

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,28 @@
+
+## jinja_tornado Makefile
+
```
Comment (westurner) [inline Makefile:3]: .PHONY default clean install-build-requirements build dist register release build-release

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+dist: build
+	python setup.py sdist bdist_wheel
+
+register:
+	python setup.py register
+
+release:
+	@# $(MAKE) register
+	twine upload dist/*
```
Comment (westurner) [inline Makefile:27]: https://pypi.python.org/pypi/twine does the right thing w/ pypi and HTTPS
- AFAIU the  setup.cfg/pypi.cfg/.config/pip/pip.conf  index url should be `https` before/for `setup.py register`

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
+
+## build requirements
+
+wheel
+twine
```
Comment (westurner) [inline requirements.txt]: REF: -> requirements[.-/]build.txt

Comment (westurner) [inline requirements.txt]: ... pbr:
| Pypi: https://pypi.python.org/pypi/pbr
| Docs: http://docs.openstack.org/developer/pbr/#id1 
| Docs: http://docs.openstack.org/developer/pbr/#environment-markers

extra requirements -> setup.cfg [distutils,setuptools,pbr]

#requirements[-pyN].txt

**Comment by westurner** [comment]
- https://github.com/thkang2/jinja_tornado/compare/master...westurner:master
- https://pypi.python.org/pypi/jinja_tornado v0.1.2

**Pull request closed without merging**